index.bs

```diff
@@ -928,7 +983,13 @@ Monkey-patches to the HTML standard {#html-monkey-patches}
 <div algorithm="Monkey patch to microtrask checkpoint">
     Append the following step to <a href="https://html.spec.whatwg.org/multipage/webappapis.html#perform-a-microtask-checkpoint">perform a microtask checkpoint</a>:
         [=Flush script entry point=].
+</div>
+
+<div algorithm="Monkey patch to pausing">
+    Add the following steps to <a href="https://html.spec.whatwg.org/multipage/webappapis.html#pause">pause</a>:
 
+    Prepend the following step: Let |timeBeforePause| be the [=monotonic clock/unsafe current time=].
```

Comment:
These two lines should be indented, I think? Probably with a blank line between them.

Reply:
Fixed